_partners/opster.md

```diff
@@ -0,0 +1,5 @@
+---
+name: Opster
+logo: 'https://opster.com/wp-content/uploads/2021/07/Opster-logo-for-opensearch.png'
```

Comment:
I'm going to approve this, but do you mind if I grab a version of your logo from your website without a white background and store it as part of the website repo (hot linking too fragile for my tastes).

<img width="320" alt="Screen Shot 2021-07-06 at 3 35 07 PM" src="https://user-images.githubusercontent.com/1152927/124669654-fea69c00-de6f-11eb-9423-639eb3dbb3b4.png">

Reply:
thanks @stockholmux , yes sure - please feel free to use a proper Opster logo that fit the website. you can download it and store on the website repo too. thanks

Reply:
![Opster-logo-for-opensearch](https://user-images.githubusercontent.com/9026326/124907776-3852e080-dff1-11eb-97d8-a24f80beb709.png)

@stockholmux I added the logo image without background here for your convenience - let me know if you need anything.